storage/innobase/srv/srv0srv.cc

```diff
@@ -2917,6 +2950,8 @@ DECLARE_THREAD(srv_master_thread)(
 {
 	srv_slot_t*	slot;
 	ulint		old_activity_count = srv_get_activity_count();
+	ulint		old_ibuf_merge_activity_count
+		= srv_sys->ibuf_merge_activity_count;
```

Comment:
May be add a new function
ulint srv_get_ibuf_merge_activity_count(void)
 { return(srv_sys->ibuf_merge_activity_count);  }
similar to srv_get_activity_count();